.eslintrc.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
   },
   extends: ["plugin:vue/essential", "eslint:recommended", "@vue/typescript"],
   rules: {
-    "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
```

Comment:
why this change

Reply:
is there a reason for the no console.log rule? I thought it was unnecessary. I wanted to log for successful and unsuccessful api requests

Reply:
For frontend code, logging is usually a bad idea since anyone can directly see what you logged. In the past, I see random `console.log` everywhere in the repos where this rule is disabled. It makes debugging harder since you need to find what you logged in a sea of other logs.
With respect to error handling, you should actually deal with the error instead of making it silently fail in the log. Evan making it crash is sometimes better than logging because it makes the problem more visible.